LeadAlerts.Web.Tests/Controllers/HomeControllerTest.cs

```diff
@@ -12,7 +12,7 @@ public void GivenAIndexAction_ThenRendersDefaultView()
             var controller = new HomeController();
 
             controller.WithCallTo(c => c.Index())
-                .ShouldRenderDefaultView();
+                .ShouldRenderView("Index");
```

Comment:
Why did you use `ShouldRenderView(...)`? You're calling the action method `Index`, the default view for that one is "Index". `ShouldRenderDefaultView` has the necessary semantics to express the intention of the test. What do you think?